src/discussions/comments/data/api.js

```diff
@@ -78,11 +78,13 @@ export async function postComment(comment, threadId, parentId) {
  * @param {string} comment Raw updated comment data to post.
  * @returns {Promise<{}>}
  */
-export async function updateComment(commentId, comment) {
+export async function updateComment(commentId, { voted } = {}) {
   const url = `${commentsApiUrl}${commentId}/`;
-
+  const patchData = snakeCaseObject({
+    voted,
+  });
   const { data } = await getAuthenticatedHttpClient()
-    .patch(url, { raw_body: comment }, { headers: { 'Content-Type': 'application/merge-patch+json' } });
+    .patch(url, patchData, { headers: { 'Content-Type': 'application/merge-patch+json' } });
   return data;
 }
```

Comment:
This will break existing code, since you'll no longer be able to update the comment, just mark it as voted/unvoted. 
Remove all data changes from this since this is just a presentational change.